catroid/gradle/intellij_config_tasks.gradle

```diff
@@ -29,6 +29,7 @@ task gitSkipWorktreeForIntellijConfigFiles << {
         'git update-index --skip-worktree Catroid.iml'.execute().text.trim()
         'git update-index --skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
         'git update-index --skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
+        'git update-index --skip-worktree catroid/catroid.iml'.execute().text.trim()
```

Comment:
Great!
I just assigned myself to the ticket as a placeholder, so nobody else will start working on it.